openassessment/staffgrader/serializers/assessments.py

```diff
@@ -0,0 +1,60 @@
+"""
+Serializers for assessment details for the ESG app
+"""
+
+from rest_framework import serializers
+from openassessment.assessment.models.base import Assessment, AssessmentPart
+
+
+# pylint: disable=abstract-method
+class SubmissionDetailFileSerilaizer(serializers.Serializer):
+    """
+    Serialized info about a single file
+    """
+    download_url = serializers.URLField()
+    description = serializers.CharField()
+    name = serializers.CharField()
+    size = serializers.IntegerField()
+
+
+class AssessmentPartSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(source='criterion.name')
+    option = serializers.CharField(source='option.name', default=None)
+    points = serializers.IntegerField(source='option.points', default=None)
+
+    class Meta:
+        model = AssessmentPart
+        fields = ['name', 'option', 'points', 'feedback']
```

Comment:
Where does 'feedback' come from? Same question for `AssessmentScoreSerializer`.

Reply:
In both places they come directly from the model. If you have a field on a model that's specify in fields on a ModelSerializer it will automatically include it. The other fields have to be manually defined because they're doing `source`-y stuff

Reply:
And I assume you mean `AssessmentSerializer`